src/shared/scrapers/__tests__/scrapers.all.test.js

```diff
@@ -3,14 +3,11 @@
 /* eslint-disable no-use-before-define */
 
 import { sync as glob } from 'fast-glob';
-import { readFileSync as readFile } from 'fs';
-import path from 'path';
 import join from '../../lib/join.js';
 import { readJSON } from '../../lib/fs.js';
-import { get } from '../../lib/fetch/get.js';
 import runScraper from './run-scraper.js';
 
-jest.mock('../../lib/fetch/get.js');
+// jest.mock('../../lib/fetch/get.js');
```

Comment:
Remove comment

Reply:
I had to change how that test worked in order to port this test without Jest's mocking; that comment needs to remain in place for this test to continue working.